polynomial-regression-core/ui.js

```diff
@@ -58,14 +58,14 @@
        {
          'mark': 'point',
          'encoding': {
-           'x': {'field': 'x', 'type': 'quantitative'},
+           'x': {'field': 'x', 'type': 'ordinal'},
```

Comment:
This is quantitative data, not ordinal data. Changing the x axis to ordinal results in a poorly rendered line as seen below: 

![image](https://user-images.githubusercontent.com/29467874/39332135-68e4adda-495b-11e8-903d-7bdb9d838c27.png)